index.html

```diff
@@ -17,46 +23,48 @@
                 // hint: these are strings
                 var numberOne = document.forms.inputs.numberOne.value;
                 var numberTwo = document.forms.inputs.numberTwo.value;
-                document.forms.inputs.result.value = numberOne + numberTwo;
+                document.forms.inputs.result.value = parseInt(numberOne) + parseInt(numberTwo);
             },
             subtractNumbers: function () {
-                
+                var numberOne = document.forms.inputs.numberOne.value;
+                var numberTwo = document.forms.inputs.numberTwo.value;
+                document.forms.inputs.result.value = parseInt(numberOne) - parseInt(numberTwo);
             }
         }
 
     </script>
 
 </head>
 
-<body>
-    <div>Header</div>
+<body style="background-color: rgb(148, 109,221);">
```

Comment:
Consider putting the background style into the body selector in the style tag in the header.